app/javascript/packs/application.ts

```diff
@@ -20,4 +25,4 @@ import 'stylesheets/main'
 // const images = require.context('../images', true)
 // const imagePath = (name) => images(name, true)
 
-require('modules/validate-forms')
+import * from './modules/validate-forms'
```

Comment:
My IDE complains about this. Not sure if that means I need to tweak the TS/JS support in the IDE or if it's just plain wrong. Have not tested in browser yet.